web/modules/custom/ecc_cct/js/ecc_cct.js

```diff
@@ -0,0 +1,43 @@
+
+
+(function ($, Drupal, drupalSettings, cookies) {
+
+  Drupal.behaviors.eccCCT = {
+    attach: function (context) {
+      var count = 0;
+      var postPreferencesLoadHandler = function (response) {
+        // The event usually fires twice: once on initial click, then
+        // preferences are taken into account and it fires again.
+        // We can't reasonably capture dithering users who change choices
+        // several times per page load.
+        count++;
+        if (count === 2) {
+          console.log(response);
```

Comment:
Remove the console logs or add a debug mode so we can control if they are written.

Reply:
console.log removed, PR updated.